decidim-core/app/controllers/decidim/timeouts_controller.rb

```diff
@@ -5,6 +5,8 @@
 module Decidim
   # Tells/Extends time before inactivity warning or automatic logout.
   class TimeoutsController < Decidim::ApplicationController
+    include Decidim::SkipTimeoutable
```

Comment:
Also here, remove the `prepend_before_action :skip_timeout, only: :index` once it's moved to `SkipTimeoutable` concern.